### DISCUSSION TITLE
Bring CRDT to ordinals

### DISCUSSION THREAD

**Discussion by losingle** [ordinals/ord#3657] [Ideas]
Bring CRDT to ordinals

Ordinals is most criticized in the Chinese community for its index centralization.

When I first came into contact with CRDT, I felt that this technology could be applied very efficiently to the blockchain.

Farcaster has become a blockbuster recently. I saw that their hubs design has used CRDT for message synchronization. 
For details, see [farcaster crdt](https://github.com/farcasterxyz/protocol/blob/main/docs/SPECIFICATION.md#31-crdts)
Can Ordinals use CRDT to achieve index synchronization, and combine it with libp2p to make each ordinals index a mutually synchronized network?

I think ordinals should go further and implement decentralized indexing as soon as possible. 
I believe it can be solved.

**Comment by nghuyenthevinh2000** [comment]
how much criticized? If there is a movement on this, I would love to join
